src/composables/Project.js

```diff
@@ -662,6 +667,41 @@ export async function deleteProjectFile(projectId, filePath, deleteParentFolder)
   }
 }
 
+/**
+ * Delete diagram parsable file.
+ * @param {string} pluginName - Name of the plugin.
+ * @param {string} projectId - Id of project.
+ * @param {string} filePath - File path to delete.
+ * @returns {Promise<void>} Promise with nothing on success otherwise an error.
+ */
+export async function deleteDiagramFile(pluginName, projectId, filePath) {
```

Comment:
Maybe, like in the documentation, it would be good to precise in the function's name that it deletes parsable files.

Moreover, I would rename also a bit because you named it "File" but you you give a folder (since you are checking if the filepath is a folder)

Reply:
It will be added by improving the documentation